testing/coreruleset/.ftw.yml

```diff
@@ -7,5 +7,8 @@ testoverride:
     920270-4: 'Rule works, log contains 920270. Test expects status 400 (Apache behaviour)'
     920272-5: 'Rule works, log contains 920272. Test expects status 400 (Apache behaviour)'
     920290-1: 'Rule works, log contains 920290. Test expects status 400 (Apache behaviour)'
+    920290-4: 'Go/http returns 400 Bad Request: missing required Host header'
```

Comment:
We should probably provide a list of rules to remove in the http package for those that are already blocked by standard http handler.

Reply:
Where? How?